server/views/layout.njk

```diff
@@ -29,14 +29,14 @@
 {% endblock %}
 
 {% block beforeContent %}
-  <div role="navigation">
-    {% if backLinkHref %}
+  {% if backLinkHref %}
```

Comment:
Unrelated - just fixing an accessibility issue the same as identified in public service: if no back link you would end up with an empty `navigation` region which could be confusing.